app/main/views/frameworks.py

```diff
@@ -84,6 +85,11 @@ def framework_dashboard(framework_slug):
     framework_dates = content_loader.get_message(framework_slug, 'dates')
     framework_urls = content_loader.get_message(framework_slug, 'urls')
 
+    try:
+        use_of_data = content_loader.get_message(framework_slug, 'use-of-data')
```

Comment:
I think the comment in the content loader is out of date since we introduced `urls` and `dates`.

The first messages was `homepage_sidebar`, which is the only thing that `block` makes sense for.